app.js

```diff
@@ -282,3 +282,23 @@ function popAllProject() {
 window.onload = () => {
   popAllProject();
 };
+
+/* Validate form */
+function onSubmit(event) {
+  const validate = document.getElementById('email');
+  const errorMessage = document.getElementById('error');
+  const email = validate.value;
+
+  if (email.toLowerCase() !== email) {
+    event.preventDefault();
+    validate.classList.add('invalid');
+    errorMessage.classList.add('error');
+    errorMessage.innerText = 'Email should be lowercase!';
```

Comment:
[Optional] You have well implemented the form validation. Well done on that. I suggest that you use a red color for the text in order to easily let the user know that there's an issue.

![image](https://user-images.githubusercontent.com/20554583/202836308-c620e199-3bfe-4423-b607-1ccebb9769e7.png)